libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/FullName.tsx

```diff
@@ -1,26 +1,64 @@
 import { Colors } from '@monorepo/expo/shared/static';
-import { CardWrapper, Input } from '@monorepo/expo/shared/ui-components';
+import {
```

Comment:
Hi @tglaz 

I have a couple of comments: 

1. Can you reduce the spacing between the title "Full Name" and "Filling out one of the..." to probably 8px? It seems to be maybe around 16px so reduce it to half if the number is true. (Please see the references from the attachments)
2. I don't see AC No.3 is happening. Upon submit, the action doesn't take me to the error location. 

Let me know if you have any questions. Thanks!

**Screenshot**
<img src="https://github.com/user-attachments/assets/bd855f05-1741-4ae7-9fd3-9280338fe109" width="320" />

My design
<img width="320" alt="image" src="https://github.com/user-attachments/assets/bd3195b9-dfef-4cae-8ca1-3372ea89422f">

Reply:
@shiozakiyuko regarding AC 3: `I don't see AC No.3 is happening. Upon submit, the action doesn't take me to the error location.`

I'm not really sure if that's the best way to handle it. I understand the issue is that currently a user can click `Add Client` and then directly hit `Submit` and it'll error out (because no name is provided). But here are a couple of thoughts handling it, and preferably in separate tickets. 

1. If required fields are not filled then the `submit` button should be disabled
2. If `name` is required to create a Client, then let's open the form to that section right away. Currently a user can enter something else in another section, click Submit and get an error. And sure, we could then take them to the Personal Info section and tell them that they need to add the name, but perhaps we shouldn't allow the user to get there in the first place?